config/cloudwatch-exporter/config.yml

```diff
@@ -0,0 +1,35 @@
+---
+discovery:
+  exportedTagsOnMetrics:
+    ec2:
+      - Name
+    rds:
+      - Name
+  jobs:
+    - type: "ec2"
+      region: $AWS_REGION
+      searchTags:
+        - Key: deployment
+          Value: ^($DEPLOY_ENV|bosh|concourse|prometheus)$
```

Comment:
With the changes to add the `deploy_env` tag, can we now filter on that instead of this set of deployments?

Reply:
Updated.